scss/_variables.scss

```diff
@@ -1815,7 +1815,7 @@ $popover-box-shadow:                var(--#{$prefix}box-shadow) !default;
 $popover-header-font-size:          $font-size-lg !default; // Boosted mod: instead of `$font-size-base`
 $popover-header-line-height:        1.11 !default; // Boosted mod
 $popover-header-bg:                 $popover-bg !default; // Boosted mod: instead of `var(--#{$prefix}secondary-bg)`
-$popover-header-color:              $headings-color !default;
+$popover-header-color:              var(--#{$prefix}body-color) !default; // Boosted mod: instead of `$headings-color`
```

Comment:
Are you sure we don't want to keep either `$headings-color` or either create a new reusable `--bs-headings-color` so that we don't lose that layer of semantics?